SPECS/cloud-hypervisor/cloud-hypervisor.signatures.json

```diff
@@ -1,7 +1,7 @@
 {
- "Signatures": {
-  "cloud-hypervisor-32.0-cargo.tar.gz": "2dd7ca374109ba337afeb0ff95d5edac8193431ec74cdbb6b1a400c600f4d915",
-  "cloud-hypervisor-32.0.tar.gz": "b9754a5ecd26697e5416a642345b2f35f4fdc983a83d540d740978309f2eb419",
-  "config.toml": "6d2aeec19782ae17eb2708262b0a7c551db3cc36b56542abca18d577de042458"
- }
-}
\ No newline at end of file
+  "Signatures": {
+    "config.toml": "6d2aeec19782ae17eb2708262b0a7c551db3cc36b56542abca18d577de042458",
+    "cloud-hypervisor-38.0.72.tar.gz": "e6d15d99c5d9ec4bede43ef8fac971d2cc0ae49a7eafffc6ca7e5b948ed4282a",
+    "cloud-hypervisor-38.0.72-vendor.tar.gz": "505887d56fb2a5ed3e19ccba42552553f9513720a0601e6ce74429d71ea53cb2"
```

Comment:
This has been renamed from `-cargo` to `-vendor`, should also be renamed in SPEC file